fixtures/onwards.mocks.ts

```diff
@@ -112,6 +115,24 @@ const trails: TrailType[] = [
 	},
 ];
 
+const trails: TrailType[] = CAPITrails.map((thisTrail) => {
+	const design = decideDesignType(thisTrail.designType);
+	const pillar = decidePillar({ pillar: thisTrail.pillar, design });
+	return {
+		url: thisTrail.url,
+		headline: thisTrail.headline,
+		showByline: thisTrail.showByline,
+		byline: thisTrail.byline,
+		image: thisTrail.image,
+		isLiveBlog: thisTrail.isLiveBlog,
+		webPublicationDate: thisTrail.webPublicationDate,
+		mediaType: thisTrail.mediaType,
+		mediaDuration: thisTrail.mediaDuration,
+		pillar,
+		design,
+	};
+});
```

Comment:
This represents the transformation between the raw api response - with CAPIDesign strings - and the typed object used in the stack - with `Design`